### ISSUE TITLE
Typos in simple_replay_buffer.py ?

### ISSUE THREAD

**Issue by duongnhatthang** [katerakelly/oyster#12]
Is this a typo ? should you change episode_starts => _episode_starts

https://github.com/katerakelly/oyster/blob/3a4777b95e1772a74142c49a33b5ca1db768d140/rlkit/data_management/simple_replay_buffer.py#L79

**Comment by katerakelly** [comment]
Yes, this only affects the recurrent encoder option, which is currently broken. See Issue 13
